includes/abstract/feedzy-rss-feeds-admin-abstract.php

```diff
@@ -1593,11 +1615,22 @@ private function get_feed_item_filter( $sc, $sizes, $item, $feed_url, $index, $i
 		if ( empty( $item_content ) ) {
 			$item_content = esc_html__( 'Post Content', 'feedzy-rss-feeds' );
 		}
+
+		$img_style = '';
+		if ( isset( $sizes['height'] ) ) {
+			$img_style = 'height:' . $sizes['height'] . 'px;';
+			if ( isset( $sc['aspectRatio'] ) ) {
```

Comment:
if the `aspectRatio` is `'1'` let's keep the backward compatibility by providing the `width`.

```php
if ( isset( $sc['aspectRatio'] ) && '1' !== $sc['aspectRatio'] ) {
```